seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java

```diff
@@ -67,7 +67,7 @@ public class MysqlCDCIT extends TestSuiteBase implements TestResource {
     private static final String MYSQL_USER_PASSWORD = "mysqlpw";
     private static final String MYSQL_DATABASE = "mysql_cdc";
     private static final String MYSQL_DATABASE2 = "mysql_cdc2";
-    private static final MySqlContainer MYSQL_CONTAINER = createMySqlContainer(MySqlVersion.V8_0);
+    private static final MySqlContainer MYSQL_CONTAINER = createMySqlContainer(MySqlVersion.V9_0);
```

Comment:
We should keep support mysql version 8.x. Please create a new test case file with mysql version 9.0.

Reply:
ok，thanks you suggest